MasKitTests/Formatters/AppListFormatterSpec.swift

```diff
@@ -25,14 +26,14 @@ class AppListsFormatterSpec: QuickSpec {
                 expect(output) == ""
             }
             it("can format a single product") {
-                products = [SoftwareProductMock(
+                let product = SoftwareProductMock(
                     appName: "Awesome App",
                     bundleIdentifier: "",
                     bundlePath: "",
                     bundleVersion: "19.2.1",
                     itemIdentifier: 12345
-                    )]
-                let output = format(products)
+                )
+                let output = format([products])
```

Comment:
This should reference the new local `product` now.

```suggestion
                let output = format([product])
```

Reply:
I applied this change but then found that `AppListFormatter` isn't included in the Xcode build and it's addition to the list command got stepped on by a merge mistake. Fixed in #339